gunrock/app/hits/hits_problem.cuh

```diff
@@ -174,17 +195,18 @@ struct Problem : ProblemBase<_GraphT, _FLAG> {
       GUARD_CU(cub_temp_space.EnsureSize_(1, target));
       GUARD_CU(hrank_mag.EnsureSize_(1, target));
       GUARD_CU(arank_mag.EnsureSize_(1, target));
+      GUARD_CU(cur_error.EnsureSize_(1, target));
 
       // Reset data
 
-      // Initialize current hrank and arank to 1.
+      // Initialize current hrank and arank to 1/nodes .
```

Comment:
Could your comment explain why its 1/nodes?

Reply:
To match NetworkX, and I've updated the comment